packages/react/src/hooks/actions/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const ACTIONS_CHANNEL = 'ui-actions';
+export const ACTION_AUTH_SIGNOUT = 'AuthSignOut';
+
+export const ACTION_STARTED = 'Started';
+export const ACTION_FINISHED = 'Finished';
+
+export const ACTION_AUTH_SIGNOUT_STARTED = `${ACTION_AUTH_SIGNOUT}.${ACTION_STARTED}`;
```

Comment:
nit: The real string `AuthSignOut.Started` may look a bit weird. It seems `AuthSignOut_Started` has a good looking IMO

Reply:
Yeah, that makes sense to me. Will update